crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -286,7 +310,29 @@ impl SceneSpawner {
 
     /// Iterate through all instances of the provided scenes and update those immediately.
     ///
-    /// Useful for updating already spawned scene instances after their corresponding scene has been modified.
+    /// Useful for updating already spawned scene instances after their corresponding scene has been
+    /// modified.
+    pub fn update_spawned_scenes(
+        &mut self,
+        world: &mut World,
+        scene_ids: &[AssetId<Scene>],
+    ) -> Result<(), SceneSpawnError> {
+        for id in scene_ids {
+            if let Some(spawned_instances) = self.spawned_scenes.get(id) {
+                for instance_id in spawned_instances {
+                    if let Some(instance_info) = self.spawned_instances.get_mut(instance_id) {
+                        Self::spawn_sync_internal(world, *id, &mut instance_info.entity_map)?;
+                    }
+                }
+            }
+        }
```

Comment:
Nothing blocking, but this could be simplified a lot by using `.filter_map` for the spawned scenes, followed by `.flat_map` for the spawned instances, followed again by `filter_map` for the instance info ;)

Reply:
I would prefer not to do this to keep it consistent with the dynamic one, and I'd prefer not to change that one to keep the diff small. bevy_scene could use a style pass IMO